enterprise/server/auth/auth.go

```diff
@@ -20,10 +23,35 @@ var (
 )
 
 func Register(ctx context.Context, env environment.Env) error {
-	authenticator, err := oidc.NewOpenIDAuthenticator(ctx, env, *adminGroupID)
+	httpAuthenticators := []interfaces.HTTPAuthenticator{}
+	userAuthenticators := []interfaces.UserAuthenticator{}
+
+	oidc, err := oidc.NewOpenIDAuthenticator(ctx, env, *adminGroupID)
 	if err != nil {
-		return status.InternalErrorf("Authenticator failed to configure: %v", err)
+		return status.InternalErrorf("OIDC authenticator failed to configure: %v", err)
+	}
+	httpAuthenticators = append(httpAuthenticators, oidc)
+	userAuthenticators = append(userAuthenticators, oidc)
+
+	if saml.SamlEnabled(env) {
```

Comment:
nit: consider renaming to `saml.IsEnabled(env)`

Reply:
Done